pkg/metrics/metrics.go

```diff
@@ -233,3 +243,11 @@ func RegisterFailedInstanceDelete(labels *MachineLabels) {
 		"reason":    labels.Reason,
 	}).Inc()
 }
+
+func DeleteMachineHealthCheckNodesCovered(name string) {
+	MachineHealthCheckNodesCovered.Delete(prometheus.Labels{"name": name})
+}
+
+func RegisterMachineHealthCheckNodesCovered(name string, count int) {
+	MachineHealthCheckNodesCovered.With(prometheus.Labels{"name": name}).Set(float64(count))
+}
```

Comment:
Nice idea on the helpers. I'd be tempted to have this and the metric altogether in the file, or maybe in a new file `machinehealthcheck.go` within this directory? Might make it easier to see the logic separation as we add more metrics over time?

Reply:
++, i like that idea